mnemosyne.yaml

```diff
@@ -0,0 +1,29 @@
+name: mnemosyne
+buildsystem: simple
+build-commands:
+  - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} .
+post-install:
+  - install -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps -Dm644 ${FLATPAK_DEST}/pixmaps/mnemosyne.png
+  - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo org.mnemosyneproj.Mnemosyne.appdata.xml
+  - install -Dm644 -t ${FLATPAK_DEST}/share/applications mnemosyne.desktop
+build-options:
+  env:
+    PYTHONUSERBASE: "${FLATPAK_DEST}"
+    PYTHONPATH: "/lib/python3.9/site-packages,${FLATPAK_DEST}/lib/python3.9/site-packages"
```

Comment:
there is no `/lib/python3.9/`. 

also PYTHONPATH separator is NOT a comma `,` but a colon `:`. In theory you shouldn't have to change it.

Reply:
would it be ok like that ?
    PYTHONPATH: "${FLATPAK_DEST}/lib/python3.9/site-packages"